### PR TITLE
PEP 8 

### PR DESCRIPTION
**Pull request by christophercalm**
Made code a little more PEP 8 compliant by adding white space. Also changed file opening code to use a with block as this closes file properly.

### CODE REVIEW

File: neoglot.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+# !/usr/bin/env python3
```
Comment (stellarskylark) [inline neoglot.py]: Remove the added space. This is not a regular comment, but a _shebang_, which tells Linux to execute it with python3 when you run the file as an executable.

**Review by stellarskylark — CHANGES_REQUESTED** [review]
Thanks for that! One little thing to fix before I merge this though.

File: neoglot.py (hunk 1 above)
Comment (christophercalm) [reply neoglot.py]: I see. Thank you! I have fixed that.

**Comment by christophercalm** [comment]
Fixed the shebang. Thank you!

**Pull request merged**